feature/identity/src/main/java/com/revenuecat/purchases/identity/IdentityManager.kt

```diff
@@ -18,7 +21,9 @@ class IdentityManager(
     private val deviceCache: DeviceCache,
     private val subscriberAttributesCache: SubscriberAttributesCache,
     private val subscriberAttributesManager: SubscriberAttributesManager,
-    private val backend: Backend
+    private val backend: Backend,
+    private val eTagManager: ETagManager,
```

Comment:
I'm doing this in iOS to validate if this abstraction makes sense (putting this in `IdentityManager`) have an improvement suggestion: you can call `backend.clearCaches` instead of injecting `ETagManager` here.

Reply:
And maybe instead of injecting the verification mode, we could expose it from `Backend`?

Reply:
Thanks for the suggestions! I liked them a lot so applied in https://github.com/RevenueCat/purchases-android/pull/844/commits/fdb918a2defdcd751a0c6f20c146688e12f9653e